From 9193c031beb092234204ddcb7bb88a2ff1c9fd99 Mon Sep 17 00:00:00 2001
From: Frank Hunleth <fhunleth@troodon-software.com>
Date: Tue, 11 Sep 2018 11:54:25 -0400
Subject: [PATCH] erlang: switch to Github

This makes it possible to download patch releases in addition to major
ones.
---
 package/erlang/erlang.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/package/erlang/erlang.mk b/package/erlang/erlang.mk
index e2cd02f68d..6e6030a012 100644
--- a/package/erlang/erlang.mk
+++ b/package/erlang/erlang.mk
@@ -6,8 +6,8 @@
 
 # See note below when updating Erlang
 ERLANG_VERSION = 22.2
-ERLANG_SITE = http://www.erlang.org/download
-ERLANG_SOURCE = otp_src_$(ERLANG_VERSION).tar.gz
+ERLANG_SITE = https://github.com/erlang/otp/archive
+ERLANG_SOURCE = OTP-$(ERLANG_VERSION).tar.gz
 ERLANG_DEPENDENCIES = host-erlang
 
 ERLANG_LICENSE = Apache-2.0
-- 
2.25.1

